Compiling for macOS?

Hi, the releases don't contain any macOS versions, wondering if it's compatible?

When you run `make build` it just throws an error

```
➜ make build
run --rm --user "501:20" app go build -ldflags="-s -w" -o '/build/tinifier' .
/bin/sh: run: command not found
make: *** [build] Error 127
```

Thanks!

--- a/.github/workflows/checks.yml
+++ b/.github/workflows/checks.yml
@@ -58,7 +58,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [linux, windows]
+        os: [linux, windows, darwin]
     needs: [gofmt, test]
     steps:
       - name: Set up Go

--- a/.github/workflows/checks.yml
+++ b/.github/workflows/checks.yml
@@ -78,7 +78,7 @@ jobs:
         run: go build -ldflags '-s -w' -o /tmp/tinifier .
 
       - name: Try to execute
-        if: matrix.os != 'windows'
+        if: matrix.os == 'linux'
         run: /tmp/tinifier -V && /tmp/tinifier -h
 
   image:

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -11,7 +11,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [linux, windows]
+        os: [linux, windows, darwin]
         arch: [amd64, 386]
         include:
           - os: windows

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## v1.0.2
+
+### Added
+
+- MacOS build
+
 ## v1.0.1
 
 ### Fixed

--- a/src/main.go
+++ b/src/main.go
@@ -5,7 +5,7 @@ import (
 	"os"
 )
 
-const VERSION = "1.0.1" // Do not forget update this value before new version releasing
+const VERSION = "1.0.2" // Do not forget update this value before new version releasing
 
 func main() {
 	cError := container.Invoke(func(logger ILogger, targets ITargets, col IAnsiColors) {

